webui/react/src/hooks/useSettings.ts

```diff
@@ -214,18 +210,14 @@ const useSettings = <T>(config: SettingsConfig<T>): UseSettingsReturn<T> => {
 
   const updateDB = useCallback(
     async (newSettings: Settings, oldSettings: SettingsRecord<T>) => {
-      const dbUpdates = Object.keys(newSettings).reduce<UserSettingUpdate[]>((acc, setting) => {
```

Comment:
I would avoid changing much here unless necessary since we want to deprecate `useSettings` anyway.